azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java

```diff
@@ -205,8 +205,8 @@ public AzkabanWebServer(final Props props,
     this.executorManagerAdapter = requireNonNull(executorManagerAdapter,
         "executorManagerAdapter is null.");
     this.projectManager = requireNonNull(projectManager, "projectManager is null.");
-    this.triggerManager = requireNonNull(triggerManager, "triggerManager is null.");
     this.missedSchedulesManager = requireNonNull(missedSchedulesManager, "missScheduleManager is null");
+    this.triggerManager = requireNonNull(triggerManager, "triggerManager is null.");
```

Comment:
Yes, it makes sense to the current structure.
However I think the long term vision is scheduling as a standalone micro-service, while its safe-guard missedScheduleManager is another. When communicate in between it will be a service call. Otherwise coupling both is not robust enough. That's why I think keeping them separately makes the migration more easy and step by step.